source/versioned-api/tests/crud-api-version-1-strict.yml

```diff
@@ -69,6 +69,10 @@ tests:
           pipeline: &pipeline
             - $listLocalSessions: {}
             - $limit: 1
+        expectError:
+          isError: true
+          errorContains: "stage $listLocalSessions is not allowed with 'apiStrict: true' in API Version 1"
```

Comment:
I think we should avoid `errorContains` here so that the test won't start failing when the server inevitably changes the error message for APIStrictError. Instead let's just include:
```
expectError:
  errorCodeName: APIStrictError

Reply:
Sounds good, I changed it to what you have above. I could also change the "distinct appends declared API version" test to avoid `errorContains`, too.